File: udemy/java/Tipos_dados/src/dados/tipos_dados.java
package dados;
import javax.swing.JOptionPane;
public class tipos_dados {

	public static void main(String[] args) {
		float salario;
		int idade;
		String nome;
		nome = JOptionPane.showInputDialog("Digite o seu nome: ");
		idade = Integer.parseInt(JOptionPane.showInputDialog("Digite o sua idade"));
		salario = Float.parseFloat(JOptionPane.showInputDialog("Digite o seu salario"));
		
		
		System.out.println("Seu nome: "+nome);
		System.out.println("Sua idade: "+idade);
		System.out.println("Seu salario: "+salario);
		
		JOptionPane.showMessageDialog(null, "Seu nome: "+nome);
		JOptionPane.showMessageDialog(null,"Sua idade: "+idade);
		JOptionPane.showMessageDialog(null, "Seu salario: "+salario);
		
	}

}

File: udemy/pascal/projetos/tipos_dados.pas
program tipos_dados;
var
     salario:real;
     idade:integer;
     nome:String[30];
begin
     //nome := 'Agostinho';
     //idade:= 18;
     //salario:= 2000;
     write('Digite o seu nome: ');
     readln(nome);
     write('Digite o seu salario: ');
     readln(salario);
     write('Digite a sua idade: ');
     readln(idade);



     writeln('Seu nome: ', nome);
     writeln('Sua idade: ', idade);
     writeln('Seu salario: ', salario);
     writeln('----------------fim----------------');
     readln();
end.
